File: trivially/src/main/java/trivially/ep/m4/finalized/Mult.java
package trivially.ep.m4.finalized;

import trivially.ep.m4.Exp;

public class Mult implements trivially.ep.m4.Mult {

    protected Exp left;

    public Exp getLeft() {
        return this.left;
    }

    protected Exp right;

    public Exp getRight() {
        return this.right;
    }

    public void setLeft(Exp left) { this.left = left; }
    public void setRight(Exp right) { this.right = right; }

    public Mult(Exp left, Exp right) {
        this.left = left;
        this.right = right;
    }
}

File: coco_csharp/CoCo/EP/M5/Finalized/Mult.cs
namespace EP.M5.Finalized {
    public class Mult : Exp, EP.M5.Mult<Exp> {
        private EP.Exp<Exp> left;
        private EP.Exp<Exp> right;

        public Mult(EP.Exp<Exp> _left, EP.Exp<Exp> _right) {
            this.left = _left;
            this.right = _right;
        }

        public EP.Exp<Exp> getLeft() { return this.left; }
        public void setLeft(EP.Exp<Exp> _left) { this.left = _left; }

        public EP.Exp<Exp> getRight() { return this.right; }
        public void setRight(EP.Exp<Exp> _right) { this.right = _right; }
    }
}